### ISSUE TITLE
Cityscapes Dataset Preparation

### ISSUE THREAD

**Issue by bit0123** [arthurdouillard/CVPR2021_PLOP#23]
I'm struggling to reproduce results on CityScapes dataset like others already reported the issue.
I found that in your paper you mentioned that with domain-incremental setting on CityScapes datasets you have used images from 21 cities in a particular order. However, after checking the dataset downloaded from [www.cityscapes-dataset.com](www.cityscapes-dataset.com), i only can se 18 cities in train directory.

Now my question is as followes:
1. Where do you get images from those three extra cities, i.e. frankfurt, lindau, and munster.
2. Could you please share more details on preparing CityScapes dataset

Thanks for your time.

**Comment by arthurdouillard** [comment]
Hey,

Thank you for your interest in our work.

1. Those three cities are in the val set. Note that while the original Cityscapes datasets use some cities as train, and other as val and test, I didn't: all cities belong to either train / val. The full order is given in the appendix of the paper (section A.3.). And for each city I used 80% for the training set (pick the val set from this), and 20% for the test set, see https://github.com/arthurdouillard/CVPR2021_PLOP/blob/main/dataset/cityscapes_domain.py#L165 where I fixed the random seed so that everyone will have the same split as me.
2. There shouldn't be anything special except unzipping the folder. Look at this class to see how I agglomerate the info from all cities of the original train and val sets: https://github.com/arthurdouillard/CVPR2021_PLOP/blob/main/dataset/cityscapes_domain.py#L79

Is that clearer?
